### DISCUSSION TITLE
Gruppensteuerung von Rolladen

### DISCUSSION THREAD

**Discussion by Donausteffen** [Zefau/ioBroker.jarvis#2093] [Q&A] [answered]
Hallo in die Runde, 
ich wollte mal Fragen ob sich zu dem Thema "Blind level Action" als Gruppensteuerung noch was getan hat? 
Für mich wäre es echt genial wenn ich noch die Rolläden eines ganzen Stockwerks per "Schalter" hoch und runterfahren könnte. 
Danke für ne kurze Info! 

Viele Grüße und weiter so!! Donausteffen

**Comment by mcuiobroker** [comment, marked as answer]
v3.1.2 ButtonGroupAction
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/geraet/actionelement/buttongroupaction-v3.1.x

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/geraet/actionelement/blindlevelaction#mehrere-rolllaeden-blinds-gleichzeitig-steuern
